runtime/binding-ws/src/scripts/io/aklivity/zilla/specs/binding/ws/streams/network/rfc8441/data.request.rpt

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2021-2022 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property writeMask ${http:randomBytes(4)}
+property client125 ${http:randomBytesUTF8(125)}
+
+connect "zilla://streams/net0"
+  option zilla:window 8192
+  option zilla:transmission "duplex"
+
+write zilla:begin.ext ${http:beginEx()
+                        .typeId(zilla:id("http"))
+                        .header(":method", "CONNECT")
+                        .header(":scheme", "http")
+                        .header(":authority", "localhost:8080")
+                        .header(":path", "/chat")
+                        .header(":protocol", "websocket")
+                        .header("sec-websocket-protocol", "chat, superchat")
+                        .header("sec-websocket-version", "13")
+                        .header("origin", "http://www.example.com")
+                        .build()}
+
+
+read zilla:begin.ext ${http:beginEx()
+                             .typeId(zilla:id("http"))
+                             .header(":status", "200")
+                             .header("sec-websocket-protocol", "chat")
+                             .build()}
+
+connected
+
+write [0x81 0xfd] ${writeMask}
+write option mask ${writeMask}
+write ${client125}
+write option mask [0x00 0x00 0x00 0x00]
+
+read [0x82 0x7d] ${client125}
```

Comment:
I the first octet to be `0x81` for text but the server is returning `0x82` (binary). I couldn't find any tests that demonstrate receiving plain text from the engine.

Reply:
Btw, missing newline at EOF.